com.unity.postprocessing/PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -973,7 +1001,7 @@ public void Render(PostProcessRenderContext context)
                     context.GetScreenSpaceTemporaryRT(cmd, lastTarget, 0, context.sourceFormat);
                     if (context.stereoActive && context.numberOfEyes > 1)
                     {
-                        if (context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.SinglePassInstanced)
+                        if (context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.SinglePassInstanced || context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.SinglePassMultiview)
```

Comment:
Please revert this line, see discussion ( https://github.com/Unity-Technologies/Graphics/pull/5917#discussion_r723683891)